src/components/common/SearchPannel/SearchPannel.jsx

```diff
@@ -0,0 +1,19 @@
+import './searchPannel.scss';
+
+const SearchPannel = () => {
+  return (
+    <form className="search-container" action="/search" method="GET">
```

Comment:
Discussion about wrapping search input in a form element

https://stackoverflow.com/a/51691457/10297026